server/webapp/WEB-INF/rails.new/app/assets/javascripts/log_output_transformer.js

```diff
@@ -124,23 +124,23 @@
 
           if (!currentSection.type()) {
             currentSection.assignType(prefix);
-            currentLine = writer.insertHeader(currentSection, prefix, line);
+            currentLine = writer.insertHeader(currentSection, prefix, timestamp, line);
```

Comment:
I'm finding it a bit difficult to understand what this additional does, or was it just a refactoring to cleanup some code?